fdbrpc/SimExternalConnection.actor.cpp

```diff
@@ -108,6 +108,57 @@ void MockDNS::clearMockTCPEndpoints() {
 	hostnameToAddresses.clear();
 }
 
+std::string MockDNS::toString() {
+	std::string ret;
+	int i = 0;
+	for (auto it = hostnameToAddresses.begin(); it != hostnameToAddresses.end(); ++it, ++i) {
```

Comment:
nit: using both `i` and `it` is slightly confusing. Consider just using `it` and use `it == hostnameToAddresses.begin()` to indicate the first hostname. If not, you add `;` at beginning of the current substring.

Not a big deal though.